modules/setting/cron.go

```diff
@@ -4,8 +4,26 @@
 
 package setting
 
+import "reflect"
+
 // GetCronSettings maps the cron subsection to the provided config
 func GetCronSettings(name string, config interface{}) (interface{}, error) {
-	err := Cfg.Section("cron." + name).MapTo(config)
-	return config, err
+	if err := Cfg.Section("cron." + name).MapTo(config); err != nil {
```

Comment:
I have 0 experience in Go, but it was hard for me to read this line. I did some research and potentially this may help to avoid such init statements in if with **return** inside

https://golang.org/doc/effective_go.html#if
```
In the Go libraries, you'll find that when an if statement doesn't flow into the next statement—that is, the body ends in break, continue, goto, or return—the unnecessary else is omitted.

f, err := os.Open(name)
if err != nil {
    return err
}
codeUsing(f)
```
More to read
https://www.calhoun.io/one-liner-if-statements-with-errors/